py/bitbox02/bitbox02/bitbox02.py

```diff
@@ -268,7 +268,7 @@ def btc_sign(
         # pylint: disable=too-many-locals,no-member
 
         # Reserved for future use.
-        assert version == 1 and locktime == 0
+        assert version in (1, 2)
```

Comment:
technically locktime was not touched in this pr and the assertion should remain
![](https://www.getdigital.de/web/getdigital/gfx/products/__generated__resized/380x380/Aufkleber_Trollface.jpg)

Reply:
god damn.